src/bundle.c

```diff
@@ -272,6 +272,11 @@ int remove_bundle(const char *bundle_name)
 		goto out_free_curl;
 	}
 
+	if (!search_bundle_in_manifest(current_mom, bundle_name)) {
+		printf("Bundle name is invalid, aborting removal\n");
```

Comment:
Since the "ret" variable is not set for this error condition, can we set ret to EBUNDLE_REMOVE in this case?